bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsVoTableCatalog.scala

```diff
@@ -67,7 +67,7 @@ case class GemsVoTableCatalog(backend: VoTableBackend = ConeSearchBackend, catal
     }))
   }
 
-  private def searchCatalog(basePosition: Coordinates, criterions: List[GemsCatalogSearchCriterion], statusLogger: StatusLogger): Future[List[GemsCatalogSearchResults]] = {
+  private def searchCatalog(basePosition: Coordinates, criterions: List[GemsCatalogSearchCriterion]): Future[List[GemsCatalogSearchResults]] = {
     val queryArgs = for {
       c <- criterions
       q = CatalogQuery(basePosition, c.criterion.radiusConstraint, c.criterion.magConstraint, catalog)
```

Comment:
Not strictly part of your PR, but this can be written as a simple call to `map`, which I prefer. 

```scala
val queryArgs = criterions.map { c =>
  val q = CatalogQuery(basePosition, c.criterion.radiusConstraint, c.criterion.magConstraint, catalog)
  (q, c)
}
```